<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="10.943" tests="2" failures="2">
  <testsuite name="Root Suite" timestamp="2025-09-08T07:58:14" tests="0" file="cypress/e2e/legal_urls.feature" time="0.000" failures="0">
  </testsuite>
  <testsuite name="Simple test to check if the links for imprint, privacy policy and terms of service are displayed" timestamp="2025-09-08T07:58:14" tests="2" time="10.937" failures="2">
    <testcase name="Simple test to check if the links for imprint, privacy policy and terms of service are displayed the user opens the CAS login page and the links for imprint, privacy policy and terms of service are displayed" time="0.000" classname="the user opens the CAS login page and the links for imprint, privacy policy and terms of service are displayed">
      <failure message="Timed out retrying after 4000ms: Expected to find element: `a`, but never found it. Queried from:

              &gt; cy.get(footer)" type="AssertionError"><![CDATA[AssertionError: Timed out retrying after 4000ms: Expected to find element: `a`, but never found it. Queried from:

              > cy.get(footer)
    at Context.eval (https://192.168.56.2/__cypress/tests?p=cypress/e2e/legal_urls.feature:16684:36)
    at Registry.runStepDefininition (https://192.168.56.2/__cypress/tests?p=cypress/e2e/legal_urls.feature:8496:48)
    at Object.fn (https://192.168.56.2/__cypress/tests?p=cypress/e2e/legal_urls.feature:15695:43)
    at runStepWithLogGroup (https://192.168.56.2/__cypress/tests?p=cypress/e2e/legal_urls.feature:15168:29)
    at Context.eval (https://192.168.56.2/__cypress/tests?p=cypress/e2e/legal_urls.feature:15691:62)]]></failure>
    </testcase>
    <testcase name="Simple test to check if the links for imprint, privacy policy and terms of service are displayed the user opens the CAS login page and legal links aswell as button texts are correctly translated" time="0.000" classname="the user opens the CAS login page and legal links aswell as button texts are correctly translated">
      <failure message="Timed out retrying after 4000ms: Expected to find element: `a`, but never found it. Queried from:

              &gt; cy.get(footer)" type="AssertionError"><![CDATA[AssertionError: Timed out retrying after 4000ms: Expected to find element: `a`, but never found it. Queried from:

              > cy.get(footer)
    at Context.eval (https://192.168.56.2/__cypress/tests?p=cypress/e2e/legal_urls.feature:16690:36)
    at Registry.runStepDefininition (https://192.168.56.2/__cypress/tests?p=cypress/e2e/legal_urls.feature:8496:48)
    at Object.fn (https://192.168.56.2/__cypress/tests?p=cypress/e2e/legal_urls.feature:15695:43)
    at runStepWithLogGroup (https://192.168.56.2/__cypress/tests?p=cypress/e2e/legal_urls.feature:15168:29)
    at Context.eval (https://192.168.56.2/__cypress/tests?p=cypress/e2e/legal_urls.feature:15691:62)]]></failure>
    </testcase>
  </testsuite>
</testsuites>